src/jira/client/jira-client.ts

```diff
@@ -33,7 +33,7 @@ export interface DeploymentsResult {
 // TODO: need to type jiraClient ASAP
 export const getJiraClient = async (
 	jiraHost: string,
-	gitHubInstallationId: number,
+	gitHubInstallationId: number | undefined,
```

Comment:
Had to add this here coz we don't pass the GH installation id when we call the appProperties put/delete from gh-config-get and the Jira uninstall event. That said, it means I've also had to update the typing for batchedBulkUpdate and sub.getSingleInstallation so happy to have alternative approaches thrown at me.

Reply:
mmmmmm this is tricky isn't it, I don't mind too much but then it forces you to update subsequent functions with the same` | undefined` which become more of a pain

other options off the top my head.  Not necessarily better mind you
- if we can, just pass the githubinstallationId anyway?
- pass -1 as the installation Id to get around it and add come comments for the magical number

Reply:
Is that -1 a little hacky though? I wasn't passing it coz at the points where it's being called it's not available e.g. in the uninstall event, that prop doesn't exist on anything we have access to. At least I don't think it does 🤔

Reply:
sweet, leave it as it is and can deal with it if the need arises